spec/url-handler-registry-spec.js

```diff
@@ -0,0 +1,75 @@
+/** @babel */
+
+import url from 'url'
+
+import {it} from './async-spec-helpers'
+
+import UrlHandlerRegistry from '../src/url-handler-registry'
```

Comment:
Some deep code review insight... To blend in with other names containing initialisms in the Atom codebase, URL should be capitalized.